src/integrationTest/java/org/opensearch/security/IpBruteForceAttacksPreventionTests.java

```diff
@@ -49,14 +56,18 @@ public class IpBruteForceAttacksPreventionTests {
     public static final String CLIENT_IP_7 = "127.0.0.7";
     public static final String CLIENT_IP_8 = "127.0.0.8";
     public static final String CLIENT_IP_9 = "127.0.0.9";
+    public static final String CLIENT_IP_10 = "127.0.0.10";
+    public static final String CLIENT_IP_11 = "127.0.0.11";
+    public static final String CLIENT_IP_12 = "127.0.0.12";
 
-    static final AuthFailureListeners listener = new AuthFailureListeners().addRateLimit(
```

Comment:
Manual backport was required because there was a difference on this line between main and 2.x. Main already had `protected` here, but 2.x did not.